.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+Project Link
```

Comment:
This is unrelated to the above, but in this case its fine to sneak it in.

In the future these should have dedicated tasks / MRs associated for easier tracking